nuxt.config.js

```diff
@@ -89,7 +89,9 @@ export default {
           info: '#B3E5FC',
           warning: '#FFF59D',
           error: '#EF9A9A',
-          accent: '#faa52e'
+          accent: '#faa52e',
+          header: '#BDBDBD',
+          menu: '#424242'
```

Comment:
This got me thinking to removing these colours (`header` and `menu`). 

Mainly because these are colours meant to be used for the material design concepts (ie. colours to provide context for primary, secondary, or success content; the ones previously listed). Also it's meant for Vuetify components that support `color=` attribute. 

So a "header" colour theme doesn't quite fit in the material design; if it's just styling certain headers, we can make a custom class instead.

Let's remove this

Reply:
The menu color theme here is being used to change the color of the menu text (buttons and dropdowns) from black to a dark grey. Should the text color for these elements still remain dark grey?

Reply:
+1 for dark grey. But if the background colour is changing for the header/menu, a different colour will be needed (white text on a blue background?).
Use the colour class instead for this colour change: https://vuetifyjs.com/en/styles/colors/#classes
Or specify a custom class in the `<style scoped>` section below in `default.vue`

Reply:
Ok. Also should the dropdown menu colors and footer colors change to match the new button/app bar colors? Currently they are both black and white.